packages/html/ds/src/select/select.schema.ts

```diff
@@ -0,0 +1,46 @@
+import * as zod from 'zod';
+import { errorTextSchema } from '../error-text/error-text.schema';
+import { hintTextSchema } from '../hint-text/hint-text.schema';
+import { labelSchema } from '../label/label.schema';
+
+export const selectSchema = zod.object({
+  id: zod.string({
+    description: 'A unique ID for the select component',
+    required_error: 'A value is required',
+  }),
+  label: labelSchema.describe('Properties for the label component').optional(),
+  options: zod
+    .object({
+      default: zod
```

Comment:
Can `default` be optional?

Reply:
Fixed. The default is now the first item in the options array